devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy

```diff
@@ -204,7 +204,7 @@ def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulat
 }
 
 def isFibaro() {
-	(zwaveInfo?.mfr == "010F" && zwaveInfo.prod == "0C02")
+	(zwaveInfo?.mfr? == "010F" && zwaveInfo?.prod? == "0C02")
```

Comment:
I suspect this *might* fail in the same way that the other DTH did? Though I am not sure why just having the original line would cause a NPE? It seems that `if (null != "some string")` works in a Groovy console. Unless there is something else going on with the way `getZwaveInfo` in DM works?